packages/kanki/kanki/main.js

```
@@ -0,0 +1,1113 @@
+    timesViewed: 0,
+    lastViewed: null
+  };
+}
+
+// Default deck with words from vocabulary.js
+function createDefaultDeck() {
+  var deck = createDeck();
+  
+  if (typeof VOCABULARY !== 'undefined') {
+    for (var level in VOCABULARY) {
+      if (VOCABULARY.hasOwnProperty(level)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)